docs/example-scenario/apps/devops-with-aks.md

```diff
@@ -38,8 +38,8 @@ This scenario covers a DevOps pipeline for a Node.js web application and databas
 ### Components
 
 * [Jenkins][jenkins] is an open-source automation server that can integrate with Azure services to enable continuous integration (CI) and continuous deployment (CD). In this scenario, Jenkins orchestrates the creation of new container images based on commits to source control, pushes those images to Azure Container Registry, then updates application instances in Azure Kubernetes Service.
-* [Azure Linux Virtual Machines][azurevm-docs] are used to run the Jenkins and Grafana instances.
-* [Azure Container Registry][azureacr-docs] stores and manages container images that are used by the Azure Kubernetes Service cluster. Images are securely stored, and can replicated to other regions by the Azure platform to speed up deployment times.
+* [Azure Linux Virtual Machines][azurevm-docs] is used to run the Jenkins and Grafana instances.
```

Comment:
I think this should remain "are":  "Azure Linux Virtual Machines are used..."

Reply:
Or just strike "are used to".

Reply:
updated so the cases is consistent across the bullets [uses is]; but it flows better.